FIAT/__init__.py

```diff
@@ -31,6 +31,8 @@
 from FIAT.morley import Morley
 from FIAT.nedelec import Nedelec
 from FIAT.nedelec_second_kind import NedelecSecondKind
+from FIAT.powell_sabin import QuadraticPowellSabin6    # noqa: F401
+from FIAT.powell_sabin import QuadraticPowellSabin12   # noqa: F401
```

Comment:
The elements need to be registered below

Reply:
Done.